typescript/src/entities/product.test.ts

```diff
@@ -109,14 +113,105 @@ test('can list products from server', () => {
   });
 });
 
+test('can list products with options set', () => {
+  const merchi = new Merchi();
+  const options = {embed: {},
```

Comment:
I personally like indented way to format dictionary, like

```
a = {
     b: 1
     c:  2
}
   

```

Reply:
personally i hate both the style used here, and the style you've shown.

it's just a matter of whatever the current eslint config says. i should take a look at that, but not in this pr